scripts/updategoogle.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+# stop on errors (nonzero exit codes), uninitialized vars
+set -eu
+
+GOOGLE_DOMAINS_URL=https://www.google.com/supported_domains
+MANIFEST_PATH=src/manifest.json
+TEMPFILE=$(mktemp)
+
+trap 'rm $TEMPFILE' EXIT
+
+echo "fetching Google Search domains ..."
+if wget -q -T 30 -O "$TEMPFILE" -- $GOOGLE_DOMAINS_URL && [ -s "$TEMPFILE" ]; then
+	python scripts/updategoogle.py "$TEMPFILE" "$MANIFEST_PATH"
+	if cmp -s "$TEMPFILE" $MANIFEST_PATH; then
+		echo "    no Google Search domain updates"
+	else
+		cp "$TEMPFILE" $MANIFEST_PATH
+		echo "    updated Google Search domains in $MANIFEST_PATH"
+		echo "    please verify, update Google's MDFP list, and commit both!"
```

Comment:
Maybe out of scope, but we should really automate the MDFP list updates as well.

Reply:
Yeah, it seems like too much work (MDFP file doesn't seem well suited for auto updates) for not enough benefit. Easy enough to manually insert whatever domains if this list ever changes. The biggest benefit is knowing there are new domains semi-automatically (thanks to the release process checklist), and easily knowing which ones are new (thanks to `git diff`).